nemo/collections/common/data/lhotse/dataloader.py

```diff
@@ -443,6 +445,14 @@ def get_lhotse_sampler_from_config(config, global_rank, world_size, tokenizer=No
     cuts, use_iterable_dataset = read_cutset_from_config(config)
     use_iterable_dataset = determine_use_iterable_dataset(use_iterable_dataset, config)
 
+    if config.drop_skipme:
+        cuts = cuts.filter(lambda cut: not cut.custom.pop("_skipme", False) if hasattr(cut, "custom") else True)
```

Comment:
```suggestion
        cuts = cuts.filter(lambda cut: "_skipme" not in cut.custom if hasattr(cut, "custom") else True)
```

I suggest the following change: skip whenever this key is present. This would let us adopt the following convention: `{"_skipme": "reason for skipping"}`

Reply:
@pzelasko , for consistency in manifest entries wouldn't it be better to have "_skipme" keys for every entry with values "False" or "0" for entries we do not need to skip and "reason for skipping" for entries we need to skip?